modules/profile/manifests/tomcat/tomcat.pp

```diff
@@ -0,0 +1,59 @@
+class profile::tomcat::tomcat
```

Comment:
profile::tomcat::tomcat? really? 
Why not just profiles::tomcat?

Reply:
fixed